index.html

```diff
@@ -2301,9 +2299,187 @@ <h2>Sets and Lists</h2>
+
+<pre class="example" data-transform="updateExample"
+     data-content-type="text/turtle"
+     title="An ordered collection of values in Turtle">
+<!--
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+<http://example.org/people#joebob> foaf:nick ("joe" "bob" "jaybe") .
+-->
+</pre>
+
+<p class="changed">In JSON-LD 1.1, lists of lists, where the value of
+  a <a>list object</a>, may itself be a <a>list object</a> recusively, are
+  fully supported. For example, in <em>GeoJSON</em> (see [[RFC7946]]),
+  <em>coordinates</em> are an ordered list of <em>positions</em>, which are
+  represented as an array of two or more numbers. For example:</p>
+
+<pre class="example changed"
+     data-content-type="application/json"
+     title="Coordinates expressed in GeoJSON">
+{
+  "type": "Feature",
+  "bbox": [-10.0, -10.0, 10.0, 10.0],
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+        [
+            [-10.0, -10.0],
+            [10.0, -10.0],
+            [10.0, 10.0],
+            [-10.0, -10.0]
+        ]
+    ]
+  }
+  //...
+}
+</pre>
+
+<p class="changed">For this example, it's important that the values
+  expressed within coordinates maintain their order, which requires the use of
+  embedded list structures. In JSON-LD 1.1, we can express this using recursive
+  lists, by simply adding the appropriate context definion:</p>
+
+<pre class="example changed"
+     data-content-type="application/json"
+     data-transform="updateExample"
+     title="Coordinates expressed in JSON-LD">
+{
+  "@context": {
+    "@vocab": "https://purl.org/geojson/vocab#",
+    "type": "@type",
+    "bbox": {"@container": "@list"},
+    "coordinates": {"@container": "@list"}
+  },
+  "type": "Feature",
+  "bbox": [-10.0, -10.0, 10.0, 10.0],
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+        [
+            [-10.0, -10.0],
+            [10.0, -10.0],
+            [10.0, 10.0],
+            [-10.0, -10.0]
+        ]
+    ]
+  }
+  ####//...####
+}
+</pre>
+
+<p>This is equivalent to the expanded form, which uses <a>list objects</a>:</p>
+
+<pre class="example changed"
+     data-content-type="application/json"
+     data-transform="updateExample"
+     title="Coordinates expressed in JSON-LD (expanded)">
+[{
+  "@type": ["https://purl.org/geojson/vocab#Feature"],
+  "https://purl.org/geojson/vocab#bbox": [{
+    "@list": [
+      {"@value": -10.0},
+      {"@value": -10.0},
+      {"@value": 10.0},
+      {"@value": 10.0}
+    ]
+  }],
+  "https://purl.org/geojson/vocab#geometry": [{
+    "@type": ["https://purl.org/geojson/vocab#Polygon"],
+    "https://purl.org/geojson/vocab#coordinates": [{
+      "@list": [{
+        "@list": [
+          {"@list": [{"@value": -10.0}, {"@value": -10.0}]},
+          {"@list": [{"@value": 10.0}, {"@value": -10.0}]},
+          {"@list": [{"@value": 10.0}, {"@value": 10.0}]},
+          {"@list": [{"@value": -10.0}, {"@value": -10.0}]}
```

Comment:
This is fine. Tagential question: is there a way to indicate to JSON-LD users that the first element of these 2-tuples is longitude and the second is latitude?

Reply:
Not really. If the vocabulary were modeled in OWL, there may be some way to do this at that level, @iherman may know better.

Reply:
To be honest, even in OWL I don't see a way to do that.

Reply:
@sgillies FWIW, we've got an issue to discuss/address that. See: #7 "support @values for describing multidimensional containers"

Reply:
From a graph modeling perspective, the fact that the lat/long were originally presented in a list isn't important, is it? Only the "outer" list of coordinates has value to the graph (afaict).

Reply:
What @gkellogg and @iherman said. In JSON-LD it will just be surface syntax of arrays. At the RDF level, it will be nested rdf:Lists with no further semantics. If more semantics are required, then a different structure would also be required.